probatus/stat_tests/psi.py

```diff
@@ -36,23 +36,30 @@ def psi(d1, d2, verbose=False):
     """
     Calculates the Population Stability Index.
 
-    A simple statistical test that quantifies the similarity of two distributions. Commonly used in the banking / risk
-    modeling industry. Only works on categorical data or bucketed numerical data. Distributions must be binned/bucketed
-    before passing them to this function. Bin boundaries should be same for both distributions. Distributions must have
-    same number of buckets. Note that the PSI varies with number of buckets chosen (typically 10-20 bins are used).
+    A simple statistical test that quantifies the similarity of two distributions.
+    Commonly used in the banking / risk modeling industry.
+    Only works on categorical data or bucketed numerical data.
+    Distributions must be binned/bucketed before passing them to this function.
+    Bin boundaries should be the same for both distributions.
+    Distributions must have the same number of buckets.
```

Comment:
Can you add some link to a referenced page to this one?

Reply:
The link was misplaced lower, fixed now.
![image](https://user-images.githubusercontent.com/16015401/129714488-34de1d09-6b1d-42bf-8ca4-da0622f6482e.png)